src/ts/background/store/settings/actions.ts

```diff
@@ -3,19 +3,27 @@ import { LocalStore } from '../../../services/local-store'
 
 export const ACTION_TYPES = {
   CHANGE_THEME: 'CHANGE_THEME',
-  CHANGE_COLOR: 'CHANGE_COLOR'
+  CHANGE_COLOR: 'CHANGE_COLOR',
+  GET_SETTINGS: 'GET_SETTINGS'
 }
 
 export function changeTheme (theme: string): AnyAction {
   return {
     type: ACTION_TYPES.CHANGE_THEME,
-    payload: LocalStore.setValue('theme', theme)
+    payload: LocalStore.setValue({ settings: { theme: theme } }, theme)
```

Comment:
is this going to reset the whole setting?

Reply:
yes, that was wrong, i've updated the implementation.